m4_sessions/1811_inning/timprov.md

```diff
@@ -1,17 +1,25 @@
 
 ## Group Members:
-
+Tim Garrity, Nick Dambrosio, Graham Munro, Dina Caraballo, Abdulla Qudrat
 
 
 ## Name of Session: 
+TIMPROV
+
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Good developers know how to code. Great developers know how to communicate.
+It's week five - let's have some fun!
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+* Meaningful connections
+* Identity
```

Comment:
What about identity?

Reply:
I think what I was going for here is that, on an interview, or anywhere, you should unapologetically present yourself.